pkg/disk/v1alpha1/disk.go

```diff
@@ -20,8 +20,9 @@ import (
 	apis "github.com/openebs/maya/pkg/apis/openebs.io/ndm/v1alpha1"
 	oeapis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
 	"github.com/pkg/errors"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
What if we do not change this file ? Are we using this `pkg/disk/v1alpha1` elsewhere for block device requirement?

Reply:
No, since we removed disk related types in `github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1` and `clientset` so it requires  changes